app/routes/shop.py

```diff
@@ -76,6 +76,7 @@ def create_dive_shop():
 
   response = requests.post(request_url, headers=headers, json=payload)
   response.raise_for_status()
+  print(response.json())
```

Comment:
let's use `logging.info(response.json())` instead

Reply:
I just removed it entirely. Missed taking it out before committing.